src/Features/Core/Portable/GlobalSuppressions.cs

```diff
@@ -0,0 +1,8 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("CodeStyle", "IDE0055", Justification = "Code is from a source package", Scope = "type", Target = "Microsoft.SourceLink.Tools.SourceLinkMap")]
+[assembly: SuppressMessage("CodeStyle", "IDE0073", Justification = "Code is from a source package", Scope = "type", Target = "Microsoft.SourceLink.Tools.SourceLinkMap")]
```

Comment:
IDE0055 suppression here works fine, but IDE0073 doesn't, which is not surprising since its not tied to a type :(